ewallet/client.go

```diff
@@ -111,3 +163,71 @@ func (c *Client) GetPaymentStatusWithContext(ctx context.Context, data *GetPayme
 
 	return response, nil
 }
+
+// CreateEWalletCharge creates new e-wallet charge
+func (c *Client) CreateEWalletCharge(data *CreateEWalletChargeParams) (*xendit.EWalletCharge, *xendit.Error) {
+	return c.CreateEWalletChargeWithContext(context.Background(), data)
+}
```

Comment:
Do we will still need this?

Reply:
I mean, why we will still support the function without the `context` param?

Reply:
Oh I'm sorry, this is my first time using Go actually so I'm not really familiar and I'm still learning it. For this one, I just tried to follow the pattern that occurs in that file. What is the best practice for this one?

Reply:
@bxcodec We have always been supporting both option:
- a no-frill one without context
- an explicitly context-requiring one